test/eChronicler_web/controllers/journal_entry_controller_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule EChroniclerWeb.JournalEntryControllerTest do
+  use EChroniclerWeb.ConnCase
+
+  test "GET /journal_entries", %{conn: conn} do
+    conn = get(conn, "/journal_entries")
+    assert html_response(conn, 200) =~ "Journal Entries Page"
```

Comment:
Do you want to test that the journal entries are actually listed?

Reply:
I do want to! However, it seems like I'll need to figure out how to transmit the data to the test version of the GET request, because as of now, it's not seeing any element of the journal entries. Putting a pin in this for tonight and returning tomorrow.

Reply:
You don't necessarily need to test that each field is rendered correctly. You just want to make sure that the response includes every journal entry available. You could for example check that the number of journal entries in the response matches the number of journal entries in the database.